Sources/ArgumentParser/Usage/MessageInfo.swift

```diff
@@ -102,4 +102,17 @@ enum MessageInfo {
     case .validation, .other: return false
     }
   }
+
+  var exitCode: Int32 {
+    switch self {
+    case .help: return EXIT_SUCCESS
+    case .validation:
+      #if os(Linux)
+      return EXIT_FAILURE
+      #else
+      return EX_USAGE
+      #endif
```

Comment:
Do you need this platform conditional? [AFAICT](https://repl.it/repls/FlimsyUnwieldyStructure) Glibc defines `EX_USAGE` in the same way as Darwin.

Reply:
Thanks for verifying on that platform.